requirements.txt

```diff
@@ -12,8 +12,10 @@ mss==6.1.0
 pandas==2.0.0
 pygetwindow==0.0.9
 pyinstaller
+setuptools-lint
+sphinx
 pywin32==306; sys_platform == 'win32'
-git+https://github.com/abrichr/pynput.git
+-e git+https://github.com/abrichr/pynput.git#egg=pynput
```

Comment:
Checkout pynput fork in editable mode, for easy development

Reply:
Thank you!